tests/examples/test_examples.py

```diff
@@ -214,6 +214,11 @@ def _run_example(example):
 import warnings
 warnings.filterwarnings("ignore", ".*", UserWarning, "matplotlib.font_manager")
 
+def static_seed(*args, **kwargs):
+    return "abc"
+from bokeh.util import session_id
+session_id._get_random_string = static_seed
```

Comment:
This patches the ``bokeh.util.session_id._get_random_string`` at runtime to return a deterministic string that's internally used as the session_id.